module-4/task-2/panic/panic.go

```diff
@@ -1,9 +1,18 @@
 package panic
 
+import "fmt"
+
 func iWillPanic() {
 	panic("something")
 }
 
+func recoveryPanic() {
+	if err := recover(); err != nil {
+		fmt.Printf("recovery from panic - %q\n", err)
+	}
+}
+
 func catchPanic() {
+	defer recoveryPanic()
```

Comment:
it is not necessary to create new function, typically we do 
```
defer func() {
   recovery code 
} () 
```

Reply:
OK, I have considered that option, but finally I chose to wriet a function.